cmd/soroban-cli/src/contract/invoke.rs

```diff
@@ -46,6 +46,9 @@ pub struct Cmd {
     /// Output the cost execution to stderr
     #[clap(long = "cost")]
     cost: bool,
+    /// Run with an unlimited budget
+    #[clap(long = "unlimited-budget")]
```

Comment:
```suggestion
    #[clap(long = "unlimited-budget", conflicts_with = "rpc-url")]
```
This only applies for local sandbox mode at the moment, so we can reject if doing a remote invoke.